packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -313,8 +313,8 @@ class VmServiceWrapper implements VmService {
       processStackFrame(current: root, parent: null);
 
       // Build the trace events.
-      for (final sample in cpuSamples.samples) {
-        final tree = _CpuProfileTimelineTree.getTreeFromSample(sample);
+      for (final sample in cpuSamples.samples!) {
```

Comment:
make this more null safe: `cpuSamples.samples ?? <CpuSample>[]`

Reply:
done